srtcore/api.cpp

```diff
@@ -1449,10 +1449,10 @@ void CUDTUnited::checkBrokenSockets()
       {
          if (s->m_Status == SRTS_LISTENING)
          {
-            uint64_t elapsed = CTimer::getTime() - s->m_TimeStamp;
+            DurationUs elapsed = ClockSys::now() - s->m_TimeStamp;
             // for a listening socket, it should wait an extra 3 seconds
             // in case a client is connecting
-            if (elapsed < 3000000) // XXX MAKE A SYMBOLIC CONSTANT HERE!
+            if (elapsed.value < 3000000) // XXX MAKE A SYMBOLIC CONSTANT HERE!
```

Comment:
But I'm worried how it will map to performance. For that reason I also haven't made the constants of type `DurationUs`, though this would be even more readable.

Reply:
How about an operator to compare against an integer?

Reply:
Unsafe. I have made intentionally the `value` field public so that transformations like this are explicitly visible. The `elapsed < 300000` would hide the fact that this number is expressed in particular time unit. Your proposal to wrap it in `DurationUs` looks at least readable.